Tasks/PublishToPyPI/task.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "2d8a1d60-8ccd-11e7-a792-11ac56e9f553",
+  "name": "PublishToPyPI",
+  "friendlyName": "Publish to PyPI",
+  "description": "Create and upload a Python package to a Python Package Index using twine.",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "[More Information](https://aka.ms/pypipublisher)",
+  "category": "Utility",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [],
+  "version": {
+    "Major": 0,
+    "Minor": 135,
+    "Patch": 0
+  },
+  "minimumAgentVersion": "2.0.0",
+  "instanceNameFormat": "Package and publish to PyPI",
+  "inputs": [
+    {
+      "name": "serviceEndpoint",
+      "aliases": [ "pyPIConnection" ],
```

Comment:
Consider whether you find `pyPiConnection` more readable.  My brain breaks up `pyPIConnection` as "Py PIC onnection"

Reply:
Do you prefer `pyPiConnection` for `pypiConnection`?  The latter seems easiest to read to me.

Reply:
ah yes, agreed on `pypiConnection`